### PR TITLE
Adds Github issue and pull request templates

### LINKED ISSUE #140: Better labels and templates for issues needed for this repo
We probably need more labels for issues - I don't see one for `kind/doc` and some others that exist for example in https://github.com/knative/serving/labels ?

### PR DESCRIPTION
**Pull request by navidshaikh**
Fixes #140 
- [x]   We might need to add few labels for knative/client repo as mentioned in the template.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+> /kind cleanup
+-->
+
+## What version of Knative Serving?
+
+<!-- Delete all but your choice -->
+
+> 0.5.x
+> 0.6.x
+> 0.7.x
```
Comment (rhuss) [inline .github/ISSUE_TEMPLATE/bug-report.md]: There won't be 0.5 - 0.7, but also 0.1 and 0.2. What also add 'nightly'.

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+## What version of Knative Serving?
+
+<!-- Delete all but your choice -->
+
+> nightly
+> 0.5.x
+> 0.6.x
+> 0.7.x
+> 0.8.x
+> Output of git describe --dirty
```
Comment (rhuss) [inline .github/ISSUE_TEMPLATE/bug-report.md]: We are introducng knative serving as client dependency in the vendor dir, so there is a direct relationship between the Kn client version and the serving used as dependency.

I guess the question here should be about the Knative Serving version running on the cluster. I would add this to the question and remove the `Output of git describe` here.

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+## What version of Knative Client?
+
+<!-- Delete all but your choice -->
+
+> nightly
+> 0.1.x
+> 0.2.x
+> Output of git describe --dirty
+
```
Comment (rhuss) [inline .github/ISSUE_TEMPLATE/bug-report.md]: A `kn version` output would be sufficient I think.

File: .github/ISSUE_TEMPLATE/bug-report.md (hunk 2 above)
Comment (navidshaikh) [reply .github/ISSUE_TEMPLATE/bug-report.md]: sounds good, updated.

File: .github/ISSUE_TEMPLATE/ask-question.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
+---
+name: Question
+about: Ask a question about knative/client
+title: ''
+labels: kind/question
```
Comment (maximilien) [inline .github/ISSUE_TEMPLATE/ask-question.md]: Nice nice @navidshaikh. Love this.

One quick question is whether the labels should also match the labels we are planning for all issues and PRs. How do we see the current defined labels?

Comment (navidshaikh) [reply .github/ISSUE_TEMPLATE/ask-question.md]: We can see the current defined labels here https://github.com/knative/client/labels

Comment (navidshaikh) [reply .github/ISSUE_TEMPLATE/ask-question.md]: I will add the labels referenced in this PR and missing from repo.

Comment (navidshaikh) [reply .github/ISSUE_TEMPLATE/ask-question.md]: Added/updated the labels https://github.com/knative/client/labels PTAL

**Comment by rhuss** [comment]
I think its a good idea (and in accordance with what Knative serving does). Let's merge that and fine tune it later.

File: .github/ISSUE_TEMPLATE/ask-question.md (hunk 4 above)
Comment (maximilien) [reply .github/ISSUE_TEMPLATE/ask-question.md]: Thank you.

**Pull request merged**